tests/tests_deps.yml

```diff
@@ -0,0 +1,64 @@
+---
+- hosts: all localhost
+  become: true
+
+  tasks:
+    - include_role:
+        name: storage
+
+    - name: test lvm and xfs package deps
+      include_tasks: run_blivet.yml
+      vars:
+        packages_only: true
+        storage_pools:
+          - name: foo
+            type: lvm
+            state: present
+            disks: []
+            volumes:
+              - name: test1
+                type: lvm
+                fs_type: xfs
+                state: present
+                mountpoint: '/foo'
+
+    - assert:
+        that:
+                - "{{ blivet_output.packages }} == ['lvm2', 'xfsprogs']"
+                - "{{ not blivet_output.changed }}"
```

Comment:
This indentation will likely run afoul of `yamllint`, possibly `ansible-lint` - also below too

Reply:
Good call -- thanks.